cli/cli.go

```diff
@@ -116,6 +117,10 @@ func (c *CLI) Client() *hcloud.Client {
 		if c.Endpoint != "" {
 			opts = append(opts, hcloud.WithEndpoint(c.Endpoint))
 		}
+		pollInterval, _ := c.RootCommand.PersistentFlags().GetInt("poll-interval")
```

Comment:
There’s a [`GetDuration()`](https://godoc.org/github.com/spf13/pflag#FlagSet.GetDuration) you can use. Allows to specify the interval with suffixes like 1s, 500ms, etc.